src/test/scala/com/exasol/cloudetl/kafka/deserialization/DeserializationFactoryTest.scala

```diff
@@ -16,15 +16,15 @@ class DeserializationFactoryTest extends AnyFunSuite {
   }
 
   test("must not deserialize key when not specified in fieldSpecs") {
-    val properties = new KafkaConsumerProperties(Map("RECORD_FORMAT" -> "json"))
+    val properties = new KafkaConsumerProperties(Map("RECORD_FORMAT" -> "jsoN"))
```

Comment:
This mixes two test cases and makes people wonder why the typo is here. maybe somebody will fix this (as a mistake) in some next PR so I think this hould either be a separate test case or we rely on the new tests already present in src/test/scala/com/exasol/cloudetl/kafka/KafkaConsumerPropertiesTest.scala

Reply:
Nice find! Yes, I did these changes before adding config tests, going to revert them back. Yes, it definitely confuses.

Reply:
Fixed.